host/xtest/xtest_main.c

```diff
@@ -107,6 +108,9 @@ void usage(char *program)
 #endif
 #ifdef CFG_SECURE_DATA_PATH
 	printf("\t--sdp-basic [opts] Basic Secure Data Path test setup ('-h' for usage)\n");
+#endif
+#ifdef CFG_PKCS11_TA
+	printf("\t--pkcs11-1028-destroy-token-object  Execute test for pkcs11_1028 case\n");
```

Comment:
"Used internally by pkcs11_208" maybe? (to make it a bit more clear that this option is not useful for the end user)

Reply:
+1. agree, I'll add this info.